src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java

```diff
@@ -208,7 +213,15 @@ public boolean canTickModule() {
         }
 
 
+
         HeroManager hero = main.hero;
+        if (this.needRefresh && System.currentTimeMillis() - lastRepair > 5_000){
+            this.needRefresh = false;
+            if(main.config.MISCELLANEOUS.REFRESH_AFTER_REVIVE ){
+                System.out.println("Triggering refresh: refreshing after death");
+                API.handleRefresh();
+            }
```

Comment:
missin the `return false;` here?

Reply:
oh i thought you wanted me to remove that